src/components/SelectionList/CheckboxListItem.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import {View} from 'react-native';
+import _ from 'underscore';
+import PressableWithFeedback from '../Pressable/PressableWithFeedback';
+import styles from '../../styles/styles';
+import Text from '../Text';
+import {checkboxListItemPropTypes} from './selectionListPropTypes';
+import Checkbox from '../Checkbox';
+import Avatar from '../Avatar';
+import OfflineWithFeedback from '../OfflineWithFeedback';
+import useLocalize from '../../hooks/useLocalize';
+
+function CheckboxListItem({item, isFocused = false, onSelectRow = () => {}, onDismissError = () => {}}) {
```

Comment:
why would `onSelectRow` and `onDismissError` have default values? They should be mandatory, right?

Reply:
For `onSelectRow` you're right! `onDismissError` not because not all lists need to show errors. Might even be removed in the future of the refactor if not used. Fixed, thanks!